src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/EventPipeCounterPipelineSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    public class EventPipeCounterPipelineSettings : EventSourcePipelineSettings
+    {
+        public EventPipeCounterGroup[] CounterGroups { get; set; }
+        public TimeSpan RefreshInterval { get; set; }
```

Comment:
Is TimeSpan appropriate for this? I've seen the metric interval is typically an integer. And the runtime seems to treat the value as a float (see https://github.com/dotnet/runtime/blob/c67684bead32185f086c3d376c6914dfbd0d4b6a/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs#L55).